src/ck-core/ckarray.h

```diff
@@ -703,6 +703,15 @@ class CkArray : public CkReductionMgr
     }
   }
 
+ inline size_t getNumLocalElems() {
+    return localElemVec.size();
+  }
+
+  inline unsigned int getEltLocalIndex(const CmiUInt8 id) {
```

Comment:
This doesn't seem like it should ever be part of the public facing interface. How the elements are stored internally could change in the future.

Reply:
I am internally maintaining the state of the zerocopy posting at the node level and for that reason, I need to find out the local index to appropriately change the state of the corresponding element in the data structure that I am maintaining. Even if the element storage is changed in the future, as long as they are indexable from 0...n-1, this method could be appropriately changed. As far as keeping it in the public-facing API, I agree. I could change the accessing functions to friend functions if you think that's a better solution?